### ISSUE TITLE
Image Rendering: Grafana image renderer plugin/service doesn't render graph panel legends to the right

### ISSUE THREAD

**Issue by vin01** [grafana/grafana#21830]
**What happened**:

When the legends are "to the right", they get missed out from the alert images. With legends below the graph, they show up as expected. No corresponding errors in the logs.

**Environment**:
- Grafana version: 6.6.0
- Data source type & version: Prometheus, 2.15.1
- OS Grafana is installed on: Ubuntu 18.04.3 LTS
- User OS & Browser: Firefox, 72.0
- Grafana plugins: grafana-image-renderer @ 1.0.8

Images appear something like following:

![image](https://user-images.githubusercontent.com/30344579/73372268-dbf17a80-42ae-11ea-9798-d28137cb3230.png)

What I also noticed is that if I manually try to render the image using a reduced width i.e `500` in this case, legends automatically come below the graph. 

URL I used to test this: https://grafana/render/d-solo/000000003/node?orgId=1&panelId=9&render=14&from=now-5m&width=500

**Comment by aknuds1** [comment]
Do you know if this used to work in previous versions of Grafana?

**Comment by vin01** [comment]
It worked until at least `6.4.3` from what I can see.

**Comment by aknuds1** [comment]
Thanks for the confirmation @vin01.

**Comment by marefr** [comment]
If you browse https://grafana/d-solo/000000003/node?orgId=1&panelId=9&from=now-5m and make the browser window ~500px wide how does that look? Any errors logged in developer tools/console?

Can you include panel json of your failing panel? Also, how many series do you have rendered for your graph?

**Comment by vin01** [comment]
No errors on resizing the window to 500px (or less) width . Responsive adjustment happens as expected.

Series count: 7

```
{
  "datasource": "Prometheus",
  "alert": {
    "alertRuleTags": {},
    "conditions": [
      {
        "evaluator": {
          "params": [
            0.9
          ],
          "type": "gt"
        },
        "operator": {
          "type": "and"
        },
        "query": {
          "params": [
            "B",
            "1m",
            "now"
          ]
        },
        "reducer": {
          "params": [],
          "type": "avg"
        },
        "type": "query"
      }
    ],
    "executionErrorState": "alerting",
    "for": "0m",
    "frequency": "60s",
    "handler": 1,
    "message": "High CPU usage",
    "name": "CPU Usage alert",
    "noDataState": "no_data",
    "notifications": [
      {
        "uid": "000000001"
      }
    ]
  },
  "alerting": {},
  "aliasColors": {
    "AVG": "#DEDAF7"
  },
  "bars": false,
  "dashLength": 10,
  "dashes": false,
  "editable": true,
  "error": false,
  "fill": 1,
  "fillGradient": 0,
  "grid": {},
  "gridPos": {
    "h": 6,
    "w": 12,
    "x": 0,
    "y": 6
  },
  "hiddenSeries": false,
  "id": 9,
  "legend": {
    "alignAsTable": true,
    "avg": false,
    "current": false,
    "max": false,
    "min": false,
    "rightSide": true,
    "show": true,
    "sideWidth": 300,
    "total": false,
    "values": false
  },
  "lines": true,
  "linewidth": 2,
  "links": [],
  "nullPointMode": "connected",
  "options": {
    "dataLinks": []
  },
  "paceLength": 10,
  "percentage": false,
  "pointradius": 5,
  "points": false,
  "renderer": "flot",
  "seriesOverrides": [],
  "spaceLength": 10,
  "stack": false,
  "steppedLine": false,
  "targets": [
    {
      "expr": "avg by (mode, instance) (irate(node_cpu_seconds_total{mode!=\"idle\", instance=~\"$server\"}[5m]))",
      "format": "time_series",
      "hide": false,
      "interval": "",
      "intervalFactor": 4,
      "legendFormat": "{{ mode }} ({{ instance }})",
      "refId": "A",
      "step": 80
    },
    {
      "expr": "1 - avg by (instance) (irate(node_cpu_seconds_total{mode=\"idle\",}[5m])) ",
      "format": "time_series",
      "hide": true,
      "intervalFactor": 2,
      "legendFormat": "{{ instance }}",
      "refId": "B"
    }
  ],
  "thresholds": [
    {
      "colorMode": "critical",
      "fill": true,
      "line": true,
      "op": "gt",
      "value": 0.9,
      "yaxis": "left"
    }
  ],
  "timeFrom": null,
  "timeRegions": [],
  "timeShift": null,
  "title": "CPU Usage",
  "tooltip": {
    "msResolution": false,
    "shared": true,
    "sort": 0,
    "value_type": "cumulative"
  },
  "type": "graph",
  "xaxis": {
    "buckets": null,
    "mode": "time",
    "name": null,
    "show": true,
    "values": []
  },
  "yaxes": [
    {
      "format": "percentunit",
      "label": "cpu usage",
      "logBase": 1,
      "max": "1",
      "min": 0,
      "show": true
    },
    {
      "format": "short",
      "label": null,
      "logBase": 1,
      "max": null,
      "min": null,
      "show": true
    }
  ],
  "yaxis": {
    "align": false,
    "alignLevel": null
  }
}
```

Please note, panel rendering works perfectly fine in the Grafana web UI, the issue of missing legends at right only occurs in the alert images.

**Comment by marefr** [comment]
@vin01 thanks. Confirmed bug. Will try to fix and include in v6.6.1 release.
